aldryn_newsblog/tests/test_aldryn_newsblog.py

```diff
@@ -81,6 +81,20 @@ def create_article(self, content=None, **kwargs):
 
         return article
 
+    def create_tagged_articles(self, num_articles=3, tags=('tag1', 'tag2')):
+        """Create num_articles Articles for each tag"""
+        articles = {}
+        for tag_name in tags:
+            tagged_articles = []
+            for _ in range(num_articles):
```

Comment:
`_` is commonly used as alias for `ugettext` or `ugettext_lazy`, so don't use `_` as variable, maybe `z` :]